server/test/email-templates.spec.es6

```diff
@@ -3,47 +3,154 @@
 const noncommercialPermitApplicationFactory = require('./data/noncommercial-permit-application-factory.es6');
 const tempOutfitterPermitApplicationFactory = require('./data/tempoutfitter-permit-application-factory.es6');
 const emails = require('../src/email/email-templates.es6');
+const noncommController = require('../src/controllers/noncommercial.es6');
+const tempOutfitterController = require('../src/controllers/temp-outfitter.es6');
 const expect = require('chai').expect;
 require('./common.es6');
 
-describe('Email templates', () => {
-  it('should send an email when noncommercial user cancels', () => {
-    const application = noncommercialPermitApplicationFactory.create();
-    application.status = 'Cancelled';
-    application.type = 'noncommercial';
-    emails.noncommercialApplicationUserCancelled(application);
-    expect(true).to.be.true;
-  });
+// to do convert to ${type}.translateFromClientToDatabase beforeall
+describe('Special use email templates', () =>{
+  const specialUseSubject = 'An update on your recent Open Forest permit application to the Forest Service.';
+  const forestName = 'Mt.Baker - Snoqualmie National Forest';
+  const adminSubject = `A new permit application with a start date of 12/12/2018 has been submitted to the ${forestName}.`;
 
-  it('should send an email when noncommercial user cancels', () => {
-    const application = noncommercialPermitApplicationFactory.create();
-    application.status = 'Review';
-    application.type = 'noncommercial';
-    emails.noncommercialApplicationReview(application);
-    expect(true).to.be.true;
-  });
+  describe('for noncommericial apps', () => {
+    let application = {};
+    beforeEach(function () {
+      // runs before each test in this block
+      let applicationFactory = noncommercialPermitApplicationFactory.create();
+      noncommController.translateFromClientToDatabase(applicationFactory, application);
+      application.forestName = forestName;
+    });
 
-  it('should send an email when temp outfitter application is rejected', () => {
-    const application = tempOutfitterPermitApplicationFactory.create();
-    application.status = 'Cancelled';
-    application.type = 'temp-outfitter';
-    emails.tempOutfitterApplicationRejected(application);
-    expect(true).to.be.true;
-  });
+    it('should build an object of email content for noncommercial app submission to user', () => {
+      const emailContent = emails.noncommercialApplicationSubmittedConfirmation(application);
+      const specialUseSubject = 'Your Noncommercial Open Forest permit application has been submitted for review!';
+      expect(emailContent.subject).to.be.eq(specialUseSubject);
+      expect(emailContent).to.have.all.keys('to','subject', 'body', 'html');
+      
+      expect(emailContent.body.trim()).to.include('NOT APPROVED');
```

Comment:
Would it make sense to make a variable of this instead of referencing the properties and `trim()` methods several times?

```js
const body = emailContent.body.trim();
const html = emailContent.html.trim();
```

Reply:
ah that's a great idea!